internal/js-dependency-scanner/src/main/java/org/wso2/security/tools/scanner/dependency/js/constants/IssueCreatorConstants.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ *   Copyright (c) 2018, WSO2 Inc., WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ * /
+ */
+
+package org.wso2.security.tools.scanner.dependency.js.constants;
+
+/**
+ * constants related to creating an issue ticket in jira
+ */
+public class IssueCreatorConstants {
+    public static final String WSO2_JIRA_BASE_URL = "https://support-staging.wso2.com/jira/";
```

Comment:
Read all the URLs from configuration files.

Reply:
Resolved